app/javascript/components/forms/SigninForm.jsx

```diff
@@ -45,7 +45,8 @@ export default function SigninForm() {
       <Row>
         <Col>
           <BootstrapForm.Group className="mb-2" controlId="formBasicCheckbox">
```

Comment:
Can't we use the `FormControl` component?

Reply:
>I'd rather not use the Form template we have because I don't feel in control when doing so.
Oh, is there a problem with the Form framework?

Reply:
```
          <FormControl
            control={BootstrapForm.Check}
            field={fields.extend_session}
            label={fields.extend_session.label}
            type="checkbox"
            noLabel
          />
```
```
export const signinFormFields = {
  email: { ... },
  password: { ... },
  extend_session: {
    label: 'Remember me',
    controlId: 'signinFormRememberMe',
    hookForm: {
      id: 'extend_session',
    },
  },
};
```

```
const validationSchema = yup.object({
  ...,
  extend_session: yup.bool(),
});

export const signinFormConfig = {
  defaultValues: {
    email: '',
    password: '',
    extend_session: false,
  },
  resolver: yupResolver(validationSchema),
};
```

Reply:
Avoiding the use of the consistent form template can have its drawbacks.

Changing how forms and form inputs behave or appear can become somehow impossible to manage if we're not having well known templates/frameworks in use. 

React hook form as a lib offers flexibility that is good to a certain extent but flexibility without consistency becomes a problem therefore the form framework is in place to first simplify the complexity of `react-hook-form` (any lib that offers a lot of flexibility has more learning curve), ensure the correct use of the lib and offer a consistent way for working with forms and displaying them.

Reply:
IMO the template adds much more complexity to React Hook Form.

In here, for example, we will be adding multiples lines of codes, in 2 extra files, to add a checkbox in our form.

When using the template, the problem comes if something is not working the way I expect it to work.
I find it really hard to debug, expand, or dealing with the Form template.
I'm not saying the problem is the template itself, it's my experience with it.

You are absolutely right that the template should be used in this example for homogeneity, as it's already been used for the Sign In Form.

Reply:
I'll give the template another try.

Reply:
Will discuss further see what's the problem with the framework on our next meeting,
If it's causing problems we should fix it or remove it and find something else.
If it's raising complexity then differently  it has a problem that we need to address.